examples/loader-compat/rspack.config.js

```diff
@@ -1,15 +1,12 @@
 /** @type {import('@rspack/cli').Configuration} */
 const config = {
+	mode: "none",
 	entry: {
 		main: "./src/index.js"
 	},
-	builtins: {
-		html: [
-			{
-				template: "./index.html"
-			}
-		]
-	},
+	target: "node",
+	externalsType: "commonjs",
+	externals: [/Debug\/iconv.node$/],
```

Comment:
If the dependency in the example is `@node-rs/crc32`, then that part of the config for `iconv` does not make sense.

Reply:
removed, and iconv is not suitable for testing because it relys on node-gyp which most of ci machine not have